python/python/tests/test_async_client.py

```diff
@@ -7503,6 +7504,55 @@ async def test_lcs_idx(self, redis_client: GlideClient):
         with pytest.raises(RequestError):
             await redis_client.lcs_idx(key1, lcs_non_string_key)
 
+    @pytest.mark.parametrize("cluster_mode", [False])
```

Comment:
Should be ran on both cluster and standalone
```suggestion
    @pytest.mark.parametrize("cluster_mode", [True, False])
```